.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: node_js
+node_js:
+- node
+notifications:
+  email: false
+deploy:
+  provider: heroku
+  api_key:
+    secure: UBOI/3MMwCfApguv41ykkxNU742VrlUsSjYIwmYCqcSyfkhrSXN10640KeGwx59CURPlQQ7ke95iU+9haq6SUzzMDoHsDcxZU4xlApFRhGWLwQEpOlMf0dSlhLfdp25HBOlugvOx9cyr0FFETUCLAjMLmVvivdh1+1vFgw4ePE9EVFA3Brt69SYtvTftPfji/2+XDsWxnXtU+dLUm0MjVwUlyupu43dm3ROVhTuaPBRHBoKSuoi0jabPOrVG08ztTwb6srOjpJ5mXgEb7/BhPveScRaTMLGkA28AQdUJWkeXO9FD4TPmvli42wzS4kcVg4u5ul/FWDqRBiVLeY+lnjdM95QtHsvbVgD9Uq1NbqXlLA1XVKPS2CEfgHMiJnFqy4CH1EnoYAYRWHOm/GhGiNGGLnBfBYUW0yJgPAG2LFSUKt3fOPNjnMrkT0f/xVB0mf0s97gbdc6sBtW3RnJb7xgpsEFqILnKc7m/OhdvpDIAVRsvM6WkO9GQ4UGrfudqN2hXzksozfmdge9ABCKusnAl3nMcsaWFoRv2jh3GgOVgk+ntGT+S0xalHMnTiW/y/mB96QjI8ddsswU9Swbkvpd+7OkwPGeJQKYG7D0bUzkRPYeC/TTNGxEcyw+K87M3qhmEl+PEzmzETS8E3dpiV0Md2WnFV7h2jAjWrqbADGo=
+  app: som-timer
```

Comment:
That's a hawt key 🔥

Reply:
Sweet that you got this up and running so fast!1